modules/persistent-storage-csi-manila-install-driver.adoc

```diff
@@ -0,0 +1,105 @@
+====
+
+.CLI procedure
+
+To install the Manila CSI driver from the CLI, complete the following steps:
+
+. Create an object YAML file, such as `maniladriver.yaml`, to define the ManilaDriver:
++
+.Example maniladriver
+[source,yaml]
+----
+  apiVersion: csi.openshift.io/v1alpha1
+  kind: ManilaDriver
+  metadata:
+    name: cluster <1>
+----
+<1> Renaming the cluster and specifying a certain namespace are not supported functions.
+
+. Create the ManilaDriver CR object in your {product-title} cluster by specifying the file you created in the previous step:
++
+----
+$ oc create -f maniladriver.yaml
+----
+
+When the Operator installation is finished, the Manila CSI driver is deployed on {product-title} for dynamic provisioning of RWX persistent volumes on {rh-openstack-first}.
+
+.Verification steps
+. Verify that the ManilaDriver CR was created successfully by entering the following command:
++
+----
+$ oc get all -n openshift-manila-csi-driver
+----
++
+Example output:
++
+----
+NAME                                                              READY   STATUS    RESTARTS   AGE
+pod/csi-nodeplugin-nfsplugin-lzvpm                                1/1     Running   0          18h
+pod/csi-nodeplugin-nfsplugin-slvg2                                1/1     Running   0          18h
+pod/csi-nodeplugin-nfsplugin-xmps9                                1/1     Running   0          18h
+pod/openstack-manila-csi-controllerplugin-7d4f5d985b-mw4x5        3/3     Running   0          17h
+pod/openstack-manila-csi-nodeplugin-6xchs                         2/2     Running   0          18h
+pod/openstack-manila-csi-nodeplugin-bkcmz                         2/2     Running   0          18h
+pod/openstack-manila-csi-nodeplugin-rlpps                         2/2     Running   0          18h
+
+NAME                                                              DESIRED CURRENT READY UP-TO-DATE AVAILABLE NODE SELECTOR AGE
+daemonset.apps/csi-nodeplugin-nfsplugin                           3       3       3     3          3                       18h
+daemonset.apps/openstack-manila-csi-nodeplugin                    3       3       3     3          3                       18h
+
+NAME                                                              READY UP-TO-DATE AVAILABLE AGE
+deployment.apps/openstack-manila-csi-controllerplugin             1/1   1          1         18h
+
+NAME                                                              DESIRED CURRENT READY AGE
+replicaset.apps/openstack-manila-csi-controllerplugin-7d4f5d985b  1       1       1     17h
+
+----
+
+. Verify that the storage class was created successfully by entering the following command:
++
+----
+$ oc get storageclasses | grep -E "NAME|csi-manila-"
+----
++
+Example output:
++
+----
+NAME              PROVISIONER               RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION  AGE
+csi-manila-ceph   manila.csi.openstack.org  Delete          Immediate           false                 18h
```

Comment:
Let's substitute 'csi-manila-gold' for 'csi-manila-ceph'.  The 'ceph' here comes from a cut-and-paste of output from PSI cloud where they happen to have a Manila share-type called 'ceph'.  'gold' is more generic and will raise fewer questions (Manila can be backed by Ceph storage but can also be backed by other sorts of storage).